pcap-dll.rc

```diff
@@ -1,12 +1,23 @@
 #include "config.h"
```

Comment:
Why?  Why not have separate .rc files for the library and the daemon, given that they're different pieces of software?

Reply:
Well, this was just my first thought since they go hand in hand.
Two separate files it is!